packages/arb-token-bridge-ui/src/hooks/useSelectedToken.ts

```diff
@@ -159,19 +172,19 @@ export async function getUsdcToken({
   if (
     (isTokenArbitrumOneNativeUSDC(tokenAddress) && isParentChainArbitrumOne) ||
     (isTokenArbitrumSepoliaNativeUSDC(tokenAddress) &&
-      isParentChainArbitrumSepolia)
+      isParentChainArbitrumSepolia) ||
+    (isTokenMainnetUSDC(tokenAddress) && isParentChainEthereumMainnet) ||
+    (isTokenSepoliaUSDC(tokenAddress) && isParentChainSepolia)
   ) {
     let childChainUsdcAddress
     try {
-      childChainUsdcAddress = isNetwork(childChainId).isOrbitChain
-        ? (
-            await getL2ERC20Address({
-              erc20L1Address: tokenAddress,
-              l1Provider: parentProvider,
-              l2Provider: childProvider
-            })
-          ).toLowerCase()
-        : undefined
+      childChainUsdcAddress = (
+        await getL2ERC20Address({
+          erc20L1Address: tokenAddress,
+          l1Provider: parentProvider,
+          l2Provider: childProvider
+        })
+      ).toLowerCase()
```

Comment:
can just use `const` and remove the try catch block?

Reply:
also it seems redundant to call it here? why not call `getChildUsdcAddress` from `useUpdateUsdcBalances.ts` here directly?

Reply:
We cannot removed the try catch because it covers a case where usdc hasn't been bridged to an Orbit chain yet.

Also we cannot use `getChildUsdcAddress` for the entire thing because it returns USDC addresses, and we use USDC.e here. We could use it just for Orbit chains and it would yield the same result because both call `getL2ERC20Address`, but it's more confusing and error prone as we have to go through the entire method first.